omnigan/losses.py

```diff
@@ -413,9 +413,7 @@ def get_losses(opts, verbose, device=None):
     # ----------------------------------
     # -----  Discriminator Losses  -----
     # ----------------------------------
-    losses["D"]["default"] = GANLoss(
-        soft_shift=opts.dis.soft_shift, flip_prob=opts.dis.flip_prob, verbose=verbose
-    )
+    losses["D"]["p"] = HingeLoss()
```

Comment:
default did not make sense, it's the painter disc

Reply:
Hmm is `losses["D"]["default"]` not used outside of the painter? I thought it makes sense to have some standard discriminator loss

Reply:
I agree, but it's not